src/nni_manager/training_service/remote_machine/shellExecutor.ts

```diff
@@ -175,7 +175,7 @@ class ShellExecutor {
     }
 
     public getRemoteExperimentRootDir(experimentId: string): string {
-        return this.joinPath(this.tempPath, 'nni', 'experiments', experimentId);
+        return this.joinPath(this.tempPath, 'nni-experiments', experimentId);
```

Comment:
This is not NNI's local home path, need to confirm if there are others folder under /tmp/nni